draft/index.html

```diff
@@ -560,7 +560,7 @@ <h3>Reconciliation Query Responses</h3>
 	    <dt><code>type</code></dt>
 	    <dd>The types of the candidate entity;</dd>
 	    <dt><code>score</code></dt>
-	    <dd>A numeral indicating how well this candidate entity matches the query: a higher score indicates a better match;</dt>
+	    <dd>An optional numeral indicating how well this candidate entity matches the query: a higher score indicates a better match;</dt>
```

Comment:
Is the list of reconciliation candidates rank ordered? I certainly think that would be most users' expectations. The spec seems silent on the issue which becomes much more noticeable in the absence of scores.

Reply:
We might want to explicitly state that the order has a sense only if score are provided.
A service not providing any score will provide an order which is totally arbitrary. 
So I propose to rephrase your proposition as

```
If candidates are scored, Reconciliation service must sort candidates in decreasing order of relevance i.e. be sorted in decreasing score order.
```